examples/pom.xml

```diff
@@ -66,7 +66,11 @@
             <groupId>org.jboss.logmanager</groupId>
             <artifactId>jboss-logmanager</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
I don't really want to bring in a dependency on guava for just one utility method